LICENSE_HEADER

```diff
@@ -1,7 +1,9 @@
-This program and the accompanying materials are made available under the terms of the
-Eclipse Public License v2.0 which accompanies this distribution, and is available at
-https://www.eclipse.org/legal/epl-v20.html
-
-SPDX-License-Identifier: EPL-2.0
-
-Copyright Contributors to the Zowe Project.
+/*
+* This program and the accompanying materials are made available and may be used, at your option, under either:
+* * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR
+* * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*/
```

Comment:
Should we follow the JSDoc comment style we've used in the LICENSE_HEADER for other projects (like [perf-timing](https://github.com/zowe/perf-timing/blob/master/LICENSE_HEADER))?
```
/**
 * xxx
 */
```